### ISSUE TITLE
Anti-ADAMTS13 antibody

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10362]
**Preferred term label:**

**Synonyms**
Auto-antibody against A Disintegrin and Metalloprotease with ThromboSpondin type 1 repeats, member 13

**Definition (free text, please give PubMed ID)**
PMID: 28416507 :In most cases, the mechanism for ADAMTS13 severe deficiency is acquired via autoantibodies to ADAMTS13, as demonstrated by positive anti-ADAMTS13 IgG in ;75% of TTP during an acute phase. These anti-ADAMTS13 IgGs usually inhibit the proteolytic activity of ADAMTS13 toward VWF, and a significant amount of circulating ADAMTS13-specific immune complexes (ICs) have been reported in acquired forms of TTP. These mechanisms both contribute to ADAMTS13 severe deficiency.

PMID: 33540569: Immune-mediated TTP, sometimes referred to as acquired TTP, is caused by ADAMTS13 deficiency mediated by autoantibodies . iTTP is further subdivided into primary iTTP, when there is no obvious associated disorder, and secondary iTTP, when an associated condition can be identified.

PMID: 31107120: Acquired TTP is the most prevalent form of the disease (~95%) and it is mainly related to autoantibodies against ADAMTS13. Acquired TTP may occur during adulthood (90% of all TTP cases) or during childhood (10% of all TTP cases)

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Immune-mediated thrombotic thrombocytopenic purpura, ORPHA:93585

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000462
- label: Anti-ADAMTS13 antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:28416507|PMID:33540569|PMID:31107120
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against A Disintegrin and Metalloprotease with ThromboSpondin type 1 repeats, member 13 (ADAMTS13).
- comment: Immune-mediated TTP, sometimes referred to as acquired TTP, is caused by ADAMTS13 deficiency mediated by autoantibodies.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10362
- synonyms:
